systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -201,6 +202,15 @@ void setup() throws IOException {
 
         ImageBuild.buildImage(suiteStorage.getNamespaceName(), IMAGE_NAME, TIERED_STORAGE_DOCKERFILE, BUILT_IMAGE_TAG, Environment.KAFKA_TIERED_STORAGE_BASE_IMAGE);
 
+        if (cluster.isKind()) {
+            final String image = Environment.getImageOutputRegistry(
+                suiteStorage.getNamespaceName(), IMAGE_NAME, BUILT_IMAGE_TAG);
+            // Kaniko pushes the image to the local registry, but Podman stores it in an internal storage
+            // that is not directly accessible to Kind. Podman must first pull the image from the registry
+            // so that it is available in the Podman daemon's local storage before "kind load" can work.
+            Exec.exec("sudo", "podman", "pull", image);
```

Comment:
what if I will use kind with docker?

Reply:
I think I should add some wrapper to check your container runtime and base on that just use podman/docker.